actuators:
  - package: dynamixel_x_mcu_controller
    class: DynamixelXMcu::ActuatorConfigParser
    config:
      dynx_1:
        servo_id: 1
        controller_name: dynx_controller
        # zero_deg: 0.0
        cw_limit_deg: -180.0
        ccw_limit_deg: 180.0
        max_effort_value: 1000.0

controllers:
  - package: dynamixel_x_mcu_controller
    class: DynamixelXMcu::ControllerConfigParser
    config:
      dynx_controller:
        controller_id: 1
        baud_rate: 57600
        publish_hz: 10.0
